tests/functional/modules/test_zos_mvs_raw_func.py

```diff
@@ -31,6 +31,16 @@
 IDCAMS_INVALID_STDIN = " hello world #$!@%!#$!@``~~^$*%"
 
```

Comment:
it looks like all the uses of the `IDCAMS_STDIN` variable have been removed, so it's probably a good idea to remove the variable definition too (line 30)